src/pages/WelcomeScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import {useTranslation} from "react-i18next";
+import {namespaces} from "../i18n/i18n.constants";
+import LogoFesbalIcon from "../components/icons/LogoFesbalIcon";
+import AppLinkButton from "../components/atom/AppLinkButton";
+
+const WelcomeScreen = () => {
+    const {t} = useTranslation(namespaces.pages.welcomeScreen);
```

Comment:
I commented on it in another PR, but it's better if we use legible names, for example, `translate` so when you use this hook inside the component you have more context to understand what `t` does 🙂

Reply:
Thanks for bringing that up. I have pushed a new commit with updated naming to make it more legible. Let me know if there's anything else you'd like me to change slightly_smiling_face